src/main/java/com/notably/logic/commands/OpenCommand.java

```diff
@@ -14,6 +17,7 @@
     public static final String COMMAND_WORD = "open";
     public static final String COMMAND_SHORTHAND = "o";
     private AbsolutePath path;
+    private final Logger logger = LogsCenter.getLogger(getClass());
```

Comment:
Oops sorry missed this before. Just a tiny thing. Let's make this `logger` a `private static final` field since we don't need to create one for each instance of `OpenCommand`!